docs/source/concepts/PropertiesFile.rst

```diff
@@ -262,6 +262,8 @@ Plotting Settings
 +---------------------------------+------------------------------------------------------------------+---------------------+
 |``plots.ShowMinorGridlines``     |Whether to show minor gridlines on plots                          | ``On``, ``Off``     |
 +---------------------------------+------------------------------------------------------------------+---------------------+
+|``plots.font``                   |The default font for labels and titles on plots.                  |``Helvetica``        |
```

Comment:
Our docs show that the default font is Helvetica, but the template properties is blank - is this being set somewhere else then or is it the MPL default?

Reply:
The last column in this table refers to an example value not a default value.